_data/faq.yml

```diff
@@ -1254,7 +1254,72 @@
                 
+                You can however mark Adventure Lab caches as "Found" locally in c:geo (Menu XYZ). Please be aware, that this "Found" marking is lost, if you delete the Adventure Lab cache from c:geo.
+        de:
+            title: Adventure Labs, die ich schon gefunden habe, werden nicht als gefunden angezeigt!
+            content: |
+                TBC
+      - id: ALC_stage
+        en:
+            title: Why are the stages of Adventure Labs not shown?
+            content: |
+                There is no final information available from Groundspeak yet, how Adventure Labs will be implemented into their application interfaces and/or websites.
+                Therefore we decided for the time being to stick to a minimum approach, which is following the recommendation bv Groundspeak and equals the implementation of other Geocaching apps.
+                
+                If we would include all currently available data for all member types, we might later end up with removing functions, if they are no longer available by Groundspeaks future interfaces.
+        de:
+            title: Warum werden die Stationen der Adventure Labs nicht angezeigt?
+            content: |
+                TBC
+      - id: ALC_hide
+        en:
+            title: I don't want to see Adventure Lab caches in c:geo. How to disable it?
+            content: |
+                Go to c:geo Menu → Settings → Services → "Geocaching.com Adventure Lab" and deactivate it.
+        de:
+            title: Ich möchte keine Adventure Labs in c:geo sehen. Wie kann ich das abschalten?
+            content: |
+                TBC
+      - id: ALC_Future
+        en:
+            title: Will there be any additional features for Adventure Labs in c:geo in the future?
+            content: |
+                We are still evaluating how we can best include Adventure Labs into c:geo and are also in contact with Groundspeak to evaluate if we find a mode of cooperation for including Adventure Labs via their official application interface.
+                There is no final information yet, about their conditions and limitations for future access to Adventure Lab data.
+                
+                If Groundspeak offers fair conditions and an acceptable functionality range in their application interface, it might be, that its possible to provide a better implementation with more functions in c:geo.
+                It might however also end up with the current state, if we are unable to find an agreement with Groundspeak or the conditions are considered to strict by our team.
+                Even worse, it might also be possible, that we need to remove Adventure Labs from c:geo in future, if they close the public access to that information and there is no further acceptable way to access this information.  
```

Comment:
M first thought was to add something like this:

If you feel that Groundspeak should support c:geo in using ALCs in a more elaborate way, please direct your wishes to the following email address: (enter a valid Groundspeak email here).

But this might go too far...

Reply:
I think its a good idea, but maybe in a more generic way (as those bad conditions would apply to all API partners):

We think it would at least be needed to have all stages and their description available in c:geo for better planning of your geocaching trips and to further improve the interworking of c:geo with their Adventure Lab app.
If you also feel that Groundspeak should design their application interface for third party apps in a more elaborate way than just providing the header coordinates of adventure lab caches, please let them know.

Reply:
I like it ☺️

Reply:
Actually, for completeness: We are already showing more than the ogirinal Geocaching App. This app only shows the dot on the map and upon touching a pin, a toast shows up with just the title and some not so intuitive icons and. No description (which we show). The next touch immediately launches the app. So they take it real serious to more or less exclusively play that game in this app. Also, there is absolutely zero integration into the maps at www.geocaching.com,

Reply:
@baiti I know, but I wanted to not explicitly mention that here ;)